tests/security/selinux/audit2allow.pm

```diff
@@ -27,9 +27,10 @@ use version_utils qw(is_sle);
 use registration qw(add_suseconnect_product);
 
 sub run {
-    my $testfile    = "test_file";
-    my $test_module = "test_module";
-    my $audit_log   = "/var/log/audit/audit.log";
+    my $testfile        = "test_file";
+    my $test_module     = "test_module";
+    my $audit_log       = "/var/log/audit/audit.log";
```

Comment:
Some times ago there was PR on some kernel tests changing .log to .txt because .txt can be reviewed within browser
So maybe use .txt also I guess .short.txt

Reply:
Done, new verification: https://openqa.suse.de/tests/6130291#